src/nmodl/symbol.cpp

```diff
@@ -76,7 +76,7 @@ Symbol* install(char* s, int t) /* install s in the list symbol table with type
     sp->usage = 0;
     sp->araydim = 0;
     sp->discdim = 0;
-    sp->level = 100;    /* larger than any reasonable submodel level */
+    sp->level = 0;      /* 0 means not found yet, 1 means already found */
```

Comment:
Instead of ```level``` I wonder if this should be renamed ```declared```.  The NMODL language allowed declarations to occur after use. Probably not a good idea. And likely a bug in some cases

Reply:
level no longer appears. No need for declared as an error is now generated for duplicate declarations.